test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -2565,7 +2566,7 @@ public void PackCommand_PackSolution_AddsProjectRefsAsPackageRefs()
             }
         }
         
-        [PlatformTheory(Platform.Windows)]
+        [PlatformTheory(Platform.Windows, Skip = "https://github.com/NuGet/Home/issues/8453")]
```

Comment:
This issue seems to be fixed already. Can we remove this now?

Reply:
Yup, will do.